src/uu/install/src/install.rs

```diff
@@ -415,6 +395,12 @@ fn behavior(matches: &ArgMatches) -> UResult<Behavior> {
         }
     };
 
+    let context = if matches.contains_id(OPT_CONTEXT) {
+        matches.get_one::<String>(OPT_CONTEXT).cloned()
+    } else {
+        None
+    };
```

Comment:
I think you can simplify it to:
```suggestion
    let context = matches.get_one::<String>(OPT_CONTEXT).cloned();
```